### DISCUSSION TITLE
Is there a way to change the cursor in AnuPpuccin to block cursor?

### DISCUSSION THREAD

**Discussion by Drakemoor** [AnubisNekhet/AnuPpuccin#213] [Snippets]
It’s tough for me to see the thin line of the default cursor, so I want to make the caret more visible i.e. wider. I can’t seem to find a way to do this. I don’t want to change the color, just the width of the cursor.

Please help.

**Comment by syrth-io** [comment]
so unfortunately, there's no way to do this via pure CSS as of now; it looks like the only property available in CSS for a cursor caret is `caret-color`. there's a [draft standard](https://www.w3.org/TR/css-ui-4/#caret-shape) for the shape (nothing for size), but it doesn't seem to have any browser support, and the document hasn't been updated since 2021. so aside from something like Javascript, your only real options right now are changing the color and/or text size. 

it would be nice if the shape property was adopted though, because I have trouble with the thin carets too.
